echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/pipelinetriggers/eventhandlers/EventHandlerSupport.groovy

```diff
@@ -0,0 +1,18 @@
+package com.netflix.spinnaker.echo.pipelinetriggers.eventhandlers
```

Comment:
Missing header

Reply:
I heard rumors that somebody heard from one of our lawyers that we don't need to do that anymore?

I'll put one in for now but it sure would be nice if we didn't have to do these shenanigans.

Reply:
Sounds good, totally agree that I'd prefer not do do these either, but hadn't heard otherwise